packages/playground/windows/playground/MainPage.cpp

```diff
@@ -73,6 +76,10 @@ void winrt::playground::implementation::MainPage::x_entryPointCombo_SelectionCha
   }
 }
 
+void MainPage::OnNavigatedTo(const Windows::UI::Xaml::Navigation::NavigationEventArgs &e) {
```

Comment:
use xaml::Navigation::NavigationEventArgs - we build the playground app both with regular xaml, and with WinUI3.  And they use different namespaces.  The xaml alias switches between the two depending on which one we are building.